entities/Rules/index.js

```diff
@@ -152,7 +165,12 @@ module.exports = {
         notify('LIQUIDATION');
       }
     },
-    ({ notify }) => notify('CONTINUE_TURN'),
+    ({ notify }, { turnValues }) => {
+      // allow continue turn as normal if pre-emptive paying of fine
```

Comment:
What does this mean, sorry ?
"Allow the player to continue turn if ... "  -- what's the "pre-emptive paying of fine" mean, specifically ?
They payed a fine before they were fined ? :S

Reply:
There is a rule wherein - if you choose to pay a fine on your turn BEFORE you roll, if you roll doubles out of jail, you can continue to roll doubles.

BUT, if you roll doubles in jail without pre-emptively paying the fine, you don't get to roll again after you're out of jail.

You are right - I think I mis-implemented this and it should be slightly different.